src/app/shared/models/hightlight-schemas.model.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * HighlightSchemas interface
+ */
+export interface HighlightSchemas {
+  /**
+   * Freshes course schema
+   */
+  isFreshCourse: string | boolean;
```

Comment:
can it be just boolean ?

Reply:
It can be boolean only, but we should add one more the same interface with freshCourse prop is string. Therefore it's the reason to use only one interface with isFreshCourse: string | boolean;